ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml

```diff
@@ -313,13 +313,15 @@ tests:
     - as: integ-servicemesh-multicluster-2-2
       cli: latest
       commands: |
+        # temporary workaround: unset ARTIFACTS env for integ-servicemesh-multicluster
         oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_MC_POD}" \
+          bash -c \
+          "unset ARTIFACTS; \
           entrypoint \
           prow/integ-suite-kind.sh \
           --topology \
           MULTICLUSTER \
-          test.integration.servicemesh.kube
-        oc cp "${MAISTRA_NAMESPACE}"/"${MAISTRA_MC_POD}":"${ARTIFACT_DIR}"/. "${ARTIFACT_DIR}"
```

Comment:
Why did you remove the `oc cp` command?

Reply:
I added that line back in the second commit